jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -1,59 +1,46 @@
 import warnings
 
 import pytest
-import numpy as np
 
 from jdaviz.configs.cubeviz.plugins.slice.slice import Slice
 
 
 def test_slice(cubeviz_helper, spectrum1d_cube):
     app = cubeviz_helper.app
     sl = Slice(app=app)
+#    sl = cubeviz_helper.plugins['Slice']._obj
```

Comment:
Which one of these makes more sense to use here?

Reply:
I can remove the commented line if we want... I wanted to test to make sure it works both ways.  I think in the future we may want to run all tests both ways since they do go through different logic routes... but that's a story for another day.